content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst

```diff
@@ -0,0 +1,58 @@
+===================================
+EU intra-community distance selling
+===================================
+
+**Distance sales within the European Union** include cross-border sales of goods and services to a
+private consumer (B2C) in another EU Member State when the seller doesn't meet face-to-face with the
+customer. Organizations must ensure that the VAT on distance sales is paid to the Member State in
+which the goods or services are delivered.
+
+.. note::
+   This remains true even if your organization is located outside of the European Union.
+
+While this regulation mainly applies to :doc:`eCommerce <../../../../websites/ecommerce>` sales to
```

Comment:
```suggestion
While this regulation mainly applies to :doc:`eCommerce <applications/websites/ecommerce>` sales to
```

Links can be clearer and stronger if they are relative from the root content directory :) (and will stay valid, even if you move the current file).

Reply:
Well, that's what I wrote initially but then @AntoineVDV  advised otherwise (in a different pull request)

-> https://github.com/odoo/documentation/pull/1044#discussion_r658545638

Reply:
Well, he validated the opposite in https://github.com/odoo/documentation/pull/966 :thinking:.
If I remember correctly, we concluded sthg like "If there is more than three "../", then the link could be changed to an absolute one" ^^.
In this case, an absolute link is IMHO stronger than a relative one, but @AntoineVDV can have the final say :sweat_smile:

Reply:
Forget it, we will use relative links to use a clear logic :ok_hand: (After discussion with ANV ^^)

Reply:
To be clear, the logic would be: use an "absolute" path in the ref when you need to traverse the root anyway.

Although @Feyensv is right that our relative link may be less robust than an absolute one, this is no longer the case when the page you're editing is buried deep in the toctree and only has the first few nodes of its path in common with the page you're making a reference to. At this point, the slightest change in the ancestor nodes of the target page will break every single link.

In short, both approaches are valid but one works best in some cases and vice-versa. Nevertheless, we need a universal logic to handle cross-app references.